tests/api_tests/features/api_tests.feature

```diff
@@ -0,0 +1,95 @@
+@api
+Feature: Newsfeed API tests
+
+  Scenario: Check microservice status
+    When get request to /status/ is received
+    Then response status should be 200
+    And response should contain "OK" status
+
+  Scenario: Verify documentation
+    When get request to /docs/ is received
+    Then response status should be 200
+    And response should contain documentation
+
+  Scenario: Get events
+    When get request to /newsfeed/123/events/ is received
+    Then response status should be 200
+    And response should contain results list
+
+  Scenario: Add events
+    When post request to /newsfeed/123/events/ is received with body:
+      | field_1   | some_data  |
+      | field_2   | other_data |
+      | field_etc | etc_data   |
+    Then response status should be 202
+    And response should include the event
+
+  Scenario: Delete event
+    Given post request to /newsfeed/123/events/ is received with body:
+      | field_1   | some_data  |
+      | field_2   | other_data |
+      | field_etc | etc_data   |
+    When delete request to /newsfeed/123/events/4567/ is received
```

Comment:
Just a note: we will need to parse `4567` from previous request. There is no way to preset or guess event id, unfortunately.

Reply:
We can use placeholder - `{event_id}`